docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -29,6 +29,11 @@ The main features of the Media Controller API include:
 
   To [receive and handle incoming commands](#receive_custom_commands) in the server, use the `addCommandListener()` method.
 
+- Setting age an rating for currently played media
```

Comment:
Did you mean "Setting age rating for currently playing media"?

Also, please use unified word for "playing media" and "played media" between line 32, 34 and line 243.